launch_ros/launch_ros/descriptions/composable_lifecycle_node.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2025 Open Navigation LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for a description of a ComposableLifecycleNode."""
+
+from typing import List
+from typing import Optional
+
+import launch
+from launch.substitution import Substitution
+# from launch.utilities import ensure_argument_type
```

Comment:
Looks like a left over.

Reply:
Fixed. that was a copy+paste from the lifecycle node (removed from both)